gdx/src/com/badlogic/gdx/graphics/g3d/particles/influencers/SpawnInfluencer.java

```diff
@@ -68,6 +70,13 @@ public void activateParticles (int startIndex, int count) {
 			positionChannel.data[i + ParticleChannels.YOffset] = TMP_V1.y;
 			positionChannel.data[i + ParticleChannels.ZOffset] = TMP_V1.z;
 		}
+		for (int i = startIndex * rotationChannel.strideSize, c = i + count * rotationChannel.strideSize; i < c; i += rotationChannel.strideSize) {
+			controller.transform.getRotation(TMP_Q, true);
```

Comment:
When we have a controller of controllers, if the parent controller was rotated, we need to communicate the rotation to the children when they're spawned.